### PR TITLE
Fix and improvements for CheckArea and CheckGroup #45

### PR DESCRIPTION
**Pull request by pedroparra**

### CODE REVIEW

File: src/CheckArea/style.css [hunk 1]
```diff
@@ -44,6 +43,8 @@
     & .ola_check {
         display: block;
         padding: 0;
+        background: var(--white);
+        border-radius: var(--radius-small);
```
Comment (oscarotero) [inline src/CheckArea/style.css]: If we move the `background` to `.ola_check-label`, we don't need the extra border-radius here.

File: src/CheckGroup/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const CheckGroup = ({ options, value, type, onChange }) => {
   return (
     <div role="radiogroup" className="ola_checkGroup" onChange={ e => onChange(e) }>
       { options && options.map( (option, idx) => (
-        <Check key={idx} name="test" type={type} value={option.value} checked={value && value == option.value}>
+        <Check key={idx} name={name || 'checkGroup'} type={type} value={option.value} checked={value && value == option.value}>
```
Comment (oscarotero) [inline src/CheckGroup/index.js]: - Not sure if it's good idea to give a default name. Maybe make this option mandatory?
- What is the purpose of `key` property? It's not defined in `Check` component

Comment (pedroparra) [reply src/CheckGroup/index.js]: - No is the better idea, but it´s a not breaker change. 
- Key is required in maps by react: https://reactjs.org/docs/lists-and-keys.html

Comment (oscarotero) [reply src/CheckGroup/index.js]: The problem with a default name is that is shared by all `Check` in the page. If there're two `CheckGroup` at the same time, there's a conflict between them. So, if it's not mandatory, I'm inclined to make it optional, but without assign a default name.

**Pull request merged**